### DISCUSSION TITLE
MAILER_MAIL not working

### DISCUSSION THREAD

**Discussion by pless84** [plausible/analytics#2374] [Self-Hosted Support] [answered]
I'm trying to send mails through our office365 smtp and it doesn't work because Mailer_mail is still hello@plausible.local instead of the mail I put in the .env file.

How can I change that mail then?

**Comment by ruslandoga** [comment, marked as answer]
@pless84 👋

The env var seems to be called [`MAILER_EMAIL`](https://github.com/plausible/analytics/blob/a86d9697baab945f8d6534edf47b4153ea5c2823/config/runtime.exs#L77)

**Reply by pless84** [reply]
Yes that is the var I use

**Reply by ruslandoga** [reply]
What's your setup like? Can you connect to the plausible instance and check that it indeed sees that env var?

**Reply by ruslandoga** [reply]
With docker compose setup, it'd be:

```console
$ cd hosting # or wherever your docker-compose.yml for plausible is located
$ docker compose exec -ti plausible bin/plausible remote
```
```elixir
iex> System.get_env("MAILER_EMAIL")
# ...output...
iex> Application.get_env(:plausible, :mailer_email)
# ...output...
```
I'm interested in what output these commands would return.

**Reply by pless84** [reply]
I get unknown shorthand, which command did you mean?

**Reply by pless84** [reply]
![billede](https://user-images.githubusercontent.com/324145/197758811-c26d749b-f1bf-4b9a-84a8-290a2dec4e03.png)

**Reply by ruslandoga** [reply]
Your screenshot shows you are using `MAILER_MAIL`, not `MAILER_EMAIL`

**Reply by pless84** [reply]
Well that explains it, doh! My bad sorry, now it works.
